<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartupSettingsDialog</class>
 <widget class="QDialog" name="StartupSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Startup settings</string>
  </property>
  <layout class="QVBoxLayout" name="dialogLayout">
   <item>
    <layout class="QGridLayout" name="mainLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="heightfieldRowsLabel">
       <property name="text">
        <string>Heightfield Rows</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="heightfieldRowsSpinBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="heightfieldColumnsLabel">
       <property name="text">
        <string>Heightfield Columns</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="heightfieldColumnsSpinBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="heightfieldHeightLabel">
       <property name="text">
        <string>Heightfield Height</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="heightfieldHeightDoubleSpinBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
